utils/are_you_sure.go

```diff
@@ -0,0 +1,72 @@
+package utils
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+type UserInputReader interface {
+	Read(secret bool) (string, error)
+}
+
+type StdinInputReader struct{}
+
+func NewStdinInputReader() *StdinInputReader {
+	return &StdinInputReader{}
+}
+
+func (reader StdinInputReader) Read(secret bool) (string, error) {
+	if secret {
+		bytePassword, err := terminal.ReadPassword(int(syscall.Stdin))
+		if err != nil {
+			return "", err
+		}
+		response := string(bytePassword)
+		return response, nil
+
+	} else {
```

Comment:
Do you need `else` here?
It  seems like if you are inside `if`, it return value

Reply:
You're right. Nothing stopping us from just defaulting